faq/index.adoc

```diff
@@ -0,0 +1,92 @@
+
+== Where are the JREs?
+
+We have already released most of the JREs for JDK8 but naturally prioritised the full JDKs before doing those. For JDK11+ we would generally recommend using jlink (see the 
+https://bugs-stage.openjdk.java.net/browse/JDK-8200132[upstream openjdk changes to stop building JREs by default]
+) but despite that statement we are currently aiming to release them for 11 as well for continuity. We do not plan to do so for JDK16 and above. See this issue for discussion on the topic.
+
+== Where are the OpenJ9 builds?
+
+The transition to Adoptium means we have unfortunately not been able to continue to distribute builds of Eclipse OpenJ9. IBM has now taken them over and they are now available as "
+https://developer.ibm.com/languages/java/semeru-runtimes/[IBM Semeru]
+". There is no need to be concerned about the change - it is still free.
+
+== Where are the docker images?
+
+Due to the limited amount of people we have working on the project, maintaining the wide variety of docker images that we provided from AdoptOpenJDK was becoming unsustainable. For that reason we are releasing a cut down set of docker images for Temurin. For those users who want a containerised Linux distribution that we don't provide an image for, use 
+https://blog.adoptium.net/2021/08/using-jlink-in-dockerfiles/[this guide]
+.
+
+[[linuxPackages]]
+== Where are the Linux rpm/deb packages? 
+
+We are working on a solution for that at the moment with 
+https://github.com/adoptium/installer/issues/330[this issue]
+as an umbrella for the work
+
+[[apiContinuity]]
+==  Will the migration break my API links? 
+
+There is a
+https://api.adoptium.net/q/swagger-ui/[new download API]
+provided by Adoptium for downloading Eclipse Temurin builds.
+
+The good news is that it looks very like the AdoptOpenJDK one did so migration isn't too difficult for anyone currently using it.*At present* the AdoptOpenJDK API will redirect to Temurin (or Semeru) if available when you request the latest release builds. We have done this because we haven't had time to encourage the community to fully migrate and because we haven't released everything yet so there are missing builds in the Adoptium API. Note that *this will be a temporary measure* and the AdoptOpenJDK API is not expected to be retained forever to do this although the timescale for removing the redirection is still to be decided.
+
+== Can you give a talk on the project?
+
+The people involved in the project are passionate about promoting it and we are keen to find ways to promote the work we do at Adoptium and with the Temurin binaries so feel free to get in touch with us if you have a forum you wish us to participate in and we will see what we can do. In general contacting the team via Slack is the best way to engage with us.
+
+https://adoptium.net/supported_platforms.html[Supported Platforms]
```

Comment:
This 2 lines should not be part of the documentation :)

Reply:
last 2 lines removed